docs/content/developers/building-contributing.md

```diff
@@ -48,6 +50,32 @@ sudo rm /usr/local/bin/ddev
 brew link --force ddev
 ```
 
+### Linux
+
+Start by removing your current binary, for Debian flavor systems use this:
+
+```
+sudo apt remove ddev
+```
+
+Next, unzip the binary you downloaded, make it executable, and move it to a folder in your path. Check with `echo $PATH`:
+
+```
+unzip ddev.zip
+chmod +x ddev && mv ddev ~/.local/bin/ddev
+```
+
+Verify the replacement worked by running `ddev version`. The output should be something like `DDEV version v1.22.3-39-gfbb878843`, instead of the regular `DDEV version v1.22.3`.
```

Comment:
Also we can check the correct binary with `which ddev` and usually you need `source ~/.bashrc` to see the changes, but I usually just restart the terminal.

Reply:
Perfect, added.